packages/plotkicadsch/plotkicadsch.0.6.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+maintainer: "Jean-Noel Avila <jn.avila@free.fr>"
+authors: "Jean-Noel Avila <jn.avila@free.fr>"
+homepage: "https://jnavila.github.io/plotkicadsch/"
+bug-reports: "https://github.com/jnavila/plotkicadsch/issues"
+doc: "https://jnavila.github.io/plotkicadsch/index"
+synopsis: "Utilities to print and compare version of Kicad schematics"
+description: """
+Two utilities:
+ * plotkicadsch is able to plot schematic sheets to SVG files
+ * plotgitsch is able to compare git revisions of schematics
+"""
+license: "ISC"
+dev-repo: "git+https://github.com/jnavila/plotkicadsch.git"
+build: [
+  [ "dune" "subst" ] {pinned}
+  [ "dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>="4.07"}
+  "dune" {>= "1.0"}
+  "kicadsch" {= version}
+  "tyxml" {>= "4.0.0"}
+  "lwt"
+  "lwt_ppx" {build}
+  "sha"
+  "git" {>= "2.0.0"}
+  "git-unix"
+  "base64" {= "2.3.0"}
+  "patience_diff" { < "v0.12.0" }
```

Comment:
Are these constraints intended? They seem a bit restrictive, some users might be annoyed by them.

Reply:
The main users of plotkicadsch and plotgitsch are Windows users. It seems that we are stuck with Janestreet's Base 0.11.0 and base64 2.3.0 in the Windows' Opam repository. On my linux box, I need to fix them so that opam is not tempted to upgrade.

I am entirely for being able to upgrade if I can have a path for generating binaries for Windows.

Reply:
I found a repo from fdopen which seems more up to date. Will try to switch to it for windows builds.